app/models/customer.py

```diff
@@ -0,0 +1,24 @@
+from flask import current_app
+from app import db
+from datetime import datetime
+from app.models.rental import Rental
```

Comment:
even though we mention "Rental" as a string argument, we aren't using the class itself or creating instances, so we don't need this

```suggestion
from datetime import datetime
```